app/containers/Profiles/index.js

```diff
@@ -24,6 +25,11 @@ const Profiles = () => {
     }
   );
   const [deleteProfile] = useMutation(DELETE_PROFILE);
+  const location = useLocation();
+
+  useEffect(() => {
+    refetch();
+  }, [location.pathname === '/profiles']);
```

Comment:
this is going to be called an unnecessary amount of times

Reply:
Updated to only be refetched when previous route wants it to